Task1/scripts/ImagesFactory.js

```diff
@@ -0,0 +1,33 @@
+import NoImage from './NoImage';
+import RemoteImage from './RemoteImage';
+import {APP_CONSTANTS} from "./constants";
+
+/**
+ * Class representing an image
+ */
+/*FACTORY PATTERN*/
+export default class ImagesFactory {
+    /**
+     * Create an article
+     * @param {string} urlToImage - url to image
+     */
+    constructor(urlToImage) {
+        let image;
```

Comment:
Please remove logic from constructor: it should either save references or call other actions.